tests/rptest/services/redpanda.py

```diff
@@ -430,6 +430,26 @@ def signal_redpanda(self, node, signal=signal.SIGKILL, idempotent=False):
 
         node.account.signal(pid, signal, allow_fail=False)
 
+    def lsof_node(self, node: ClusterNode):
+        """
+        Get the list of open files for a running node
+        :return: yields strings
+        """
+        first = True
+        for line in node.account.ssh_capture(
+                f"lsof -nP -p {self.redpanda_pid(node)}"):
```

Comment:
I looked at using `-F` here for machine readable output, but the resulting code was nastier -- that output mode outputs a mixture of per-file and per-process lines.  There doesn't seem to be a way to get a line per file _and_ a machine readable format.

Whacky stuff like filenames with newlines can trip this up in general purpose code, but in our tests we're always looking at just the files that redpanda opens, which are boring unix paths.

Reply:
wow I just spent way too much time reading `man lsof`... `-F` is... weird